roles/ceph-client/tasks/create_users_keys.yml

```diff
@@ -4,7 +4,9 @@
   with_items: "{{ pools }}"
   changed_when: false
   failed_when: false
-  when: pools | length > 0
+  when: 
+    - pools | length > 0
+    - not create_key_file_only
```

Comment:
why not simply adding `- copy_admin_key`?

Reply:
The "create pools" and "create keys" tasks fail if the admin key is not copied. My goal is to have them not run in this scenario. Using one variable `copy admin key`, and not introducing another, as a condition of running this task, works for me. I'll update the PR.

Reply:
Sure, please notify me when it's done. Thanks!